test/jest_integration/jwt_auth.test.ts

```diff
@@ -35,7 +35,9 @@ describe('start OpenSearch Dashboards server', () => {
   const pageTitleXPath = '//*[@id="osdOverviewPageHeader__title"]';
   // Browser Settings
   const browser = 'firefox';
-  const options = new Options().headless();
+  //const options = new Options().headless();
+  const options = new Options();
```

Comment:
Can you revert this back to headless when done?

Reply:
Nice catch! I'm updating that now.

Reply:
Done!